calblueprint/siren
# Bry/styling

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] keyboard doesn't cover up the text field
 - [X] removed notifications button -- put Log Out button in settings tab
 - [X] added clickable redirection links to device's email/phone-call app
 -  [X] fixed upload screen to scroll
 - [X] _**(EDITED)**_: update button does NOT work -- this is being handled in **bry/error-alerts** branch  _(OUTDATED) fixed update button to work_


## How can the reviewer test your code? 👩‍💻
 - keyboard doesn't cover up the text field
      - you should be able to see what you're typing in: Log in, Register, Settings, Intake Form screens
 
 - removed notifications button -- put Log Out button in settings tab
      - tap on the settings button
      - Logout button should log you out
      - update button should be placed underneath the Radio content
      
 - added clickable redirection links to device's email/phone-call app
      - at the top of the Home screen
![IMG_01897643F8E0-1](https://user-images.githubusercontent.com/60284767/169627442-1cf091f8-8e72-4d6e-8320-e3b9383d8040.jpeg)
      
 -  fixed upload screen to scroll
      - have multiple cases filled out
      - check that you can scroll and view all the Doc Containers for each case
      
 - fixed update button to work
      - in the code, refractored auth.ts functions updateEmail() and updateFirebaseLanguage() to take in clientId
<img width="585" alt="Screen Shot 2022-05-20 at 5 40 09 PM" src="https://user-images.githubusercontent.com/60284767/169627416-e461ac76-7678-4656-8a02-7a906ff70d4d.png">


## Updates
- **Fixed Register Problem**: 
    - **Problem**: Registering an account would register an account in the Firebase authentication but not create a client in the database. Thus, the Home Screen would not display the user's name.
    - **Fixed Functionality**: Registering an account will now create a client in the database and the Home Screen now displays the user's name.
    - **How it was fixed:** database/auth.ts register() now uses usr.user instead of user 
<img width="409" alt="Screen Shot 2022-05-27 at 3 13 01 PM" src="https://user-images.githubusercontent.com/60284767/170795421-3a04f347-cca1-4f02-bde4-4d825ff88c9e.png">



🧜‍♀️ cc: @shannonbonet
